core-it-suite/src/test/resources/mng-7310-lifecycle-activated-in-specified-module/extension/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>mng-7310-lifecycle-activated-in-specified-module</groupId>
+  <artifactId>mng-7310-lifecycle-activated-in-specified-module</artifactId>
+  <version>0.1</version>
+
+  <packaging>maven-plugin</packaging>
+
+  <properties>
+    <mavenVersion>3.6.2</mavenVersion>
```

Comment:
When using 3.5, use the latest one.

Reply:
Where am I using 3.5? Or do you mean that I should be using the latest from 3.5?

Reply:
Typo, I mean 3.6, so use 3.6.3 here.

Reply:
Ah, I think was planning to do that, but there's no 3.6.3 for [maven-plugin-annotations](https://mvnrepository.com/artifact/org.apache.maven.plugin-tools/maven-plugin-annotations). 
There's a 3.6.4, but for maven-plugin-api [there's no 3.6.4](https://mvnrepository.com/artifact/org.apache.maven/maven-plugin-api). What would you advise?

Reply:
Maven Core != Maven Plugin Tools. The property in question is solely for Maven Core. If you have MPT as well and more than one dep, then use like everywher else, `mavenPluginToolsVersion`.

Reply:
Right, my bad! I've inlined both versions since they are both just used once. Is it okay like this?